lib/widget/queue/queue_details_window.dart

```diff
@@ -70,8 +70,6 @@ class _QueueDetailsWindowState extends State<QueueDetailsWindow> {
                               height: 25,
                               child: SvgPicture.asset(
                                 FileUtil.resolveFileTypeIconPath(dl.fileType),
-                                color: FileUtil.resolveFileTypeIconColor(
```

Comment:
Please roll this change back. Removing the color field would result in black icons which doesn't look good.

Reply:
i fix this issue